Makefile.toml

```diff
@@ -129,6 +129,12 @@ echo "🧹 Cleaning old documentation..."
 rm -rf $SCHEMA_FOLDER
 rm -rf $DOCS_FOLDER
 
+if [[ $(jq --version) != jq-1\.7* ]]
```

Comment:
I think that best would be the `jq` presence with the correct version checked in the task above (`check-prerequisites`).